neurodsp/sim/periodic.py

```diff
@@ -27,9 +27,11 @@ def sim_oscillation(n_seconds, fs, freq, cycle='sine', phase=0, **cycle_params):
     cycle : {'sine', 'asine', 'sawtooth', 'gaussian', 'exp', '2exp'}
         What type of oscillation cycle to simulate.
         See `sim_cycle` for details on cycle types and parameters.
-    phase : float, optional, default: 0
+    phase : float or str, optional, default: 0
         If non-zero, applies a phase shift to the oscillation by rotating the cycle.
-        The shift is defined as a relative proportion of cycle, between [0, 1].
+        The shift is defined as a relative proportion of cycle, between [0, 1]. A string may also
+        be passed to adjust the phase to start/end cycles on either minima (``min``) or
```

Comment:
Nitpick, but lines 32-33 are slightly differently described than what should probably be same description as in `sim_cycle` 267-268. 
I would check through and make sure anything that should be the same is the same. 
I might suggest something like:
```
If a number, the shift is defined as a relative proportion of cycle, between [0, 1]. 
If 'min' or 'max', the cycle is shifted to start at it's minima or maxima. 
```